extension/src/service/web-component-service.js

```diff
@@ -125,6 +125,7 @@ function createSessionRequest(
   commercetoolsProjectKey,
   requestObject
 ) {
+  extendRequestObjWithMetadata(requestObject, commercetoolsProjectKey)
```

Comment:
Actually line 113 is needed, otherwise no project key can be obtained in notification 
https://github.com/commercetools/commercetools-adyen-integration/pull/1061/commits/1cbd5db13773acaf827113bd4f8dab972ab659bc